soc/st/stm32/stm32f3x/Kconfig.defconfig

```diff
@@ -14,4 +14,14 @@ configdefault SHARED_INTERRUPTS
 	default y if $(dt_nodelabel_enabled,timers1) && $(dt_nodelabel_enabled,timers16)
 	default y if $(dt_nodelabel_enabled,timers1) && $(dt_nodelabel_enabled,timers17)
 
+if PM
+
+config COUNTER
+        default y
```

Comment:
Could you address CI compliance checks requests (a tab instead of space chars)?

I'm surprised you experience build issues using `configudefault` instead of `config` in this file.